.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - run: sudo apt-get install cookiecutter
+      - run: |
+          cookiecutter ../template-product-infrastructure/ \
+            --no-input \
+            --config-file \
+            ../template-product-infrastructure/cookiecutter-ci.yaml
+
+      - run: ls -lash
+      - name: Super-Linter
+        uses: github/super-linter@v3.10.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          FILTER_REGEX_INCLUDE: rpetemplate/.*
```

Comment:
this seems not to work or not included in the current version:
https://github.com/github/super-linter/issues/710

https://github.com/github/super-linter/pull/684

Will revisit this PR after the next release